### DISCUSSION TITLE
Docker install version ?

### DISCUSSION THREAD

**Discussion by elChapoSing** [KL-Psychological-Methodology/ESMira#8] [Ideas]
I think it would be great to have a docker version of the server ! one click install vs the current step by step which might not fit all configurations (e.g I do not have such a simple webserver so I would have to set one up specifically for ESMira)

**Comment by SelinaDev** [comment]
Hi! Thank you for your suggestion.
I agree that a docker version would be beneficial. We actually do have this as part of our internal list, however, there are currently other items that have higher priority for us. I have experimented with creating a docker image of ESMira in the past, but have encountered issues back then. I might need so set aside some time to make this work properly, but I currently cannot say when I will get to that.

**Reply by JodliDev** [reply]
Just a heads up: This weekend I had a motivation spike and started a Docker implementation.
So far it seems to work very well. I have to change things up, to make sure migrations are happening when the docker image was updated, but I should be able to release it for testing this week :)
